### DISCUSSION TITLE
WinUI3 should bring back binding in style setters

### DISCUSSION THREAD

**Discussion by rowan-gray** [microsoft/WindowsAppSDK#1286] [Ideas] [answered]
An aspect which is severely lacking in WinUI3 compared to WPF is the ability to bind in style setters. It makes it more difficult to port applications from WPF to WinUI3 and also makes it more difficult to make a responsive UI. It's really thrown a spanner in the works for what I'm trying to accomplish using WinUI3 and is making me consider moving back to WPF simply because it's such a good quality of life feature.

I hope that this feature can be added to WinUI3 in the future to make it comparable in quality of life to WPF.

**Comment by btueffers** [comment, marked as answer]
Hi @rgray000, thank you for taking the time to provide feedback! Feature requests for WinUI are typically made by creating [an issue](https://github.com/microsoft/microsoft-ui-xaml/issues) in their repo.
